azurerm/internal/services/network/network_connection_monitor_resource.go

```diff
@@ -328,6 +323,62 @@ func resourceArmNetworkConnectionMonitor() *schema.Resource {
 				},
 			},
 
+			"auto_start": {
+				Type:       schema.TypeBool,
+				Optional:   true,
+				Computed:   true,
+				Deprecated: "The field is not supported as the API has deprecated it",
```

Comment:
Can we modify the deprecation message a bit like:

> The field belongs to the v1 network connection monitor, which is now deprecated in favor of v2 by Azure. Please check the document (https://www.terraform.io/docs/providers/azurerm/r/network_connection_monitor.html) for the v2 properties.

Reply:
updated